prosody/rootfs/defaults/prosody.cfg.lua

```diff
@@ -287,6 +294,13 @@ log = {
 {{ end }}
 }
 
+-- Statistics Provider and 
+{{ if $PROSODY_ENABLE_METRICS }}
+statistics = "internal"
+statistics_interval = "manual"
+openmetrics_allow_cidr = "172.18.0.0/16"
```

Comment:
This should be a configurable CIDR via env variable and default to 172.16.0.0/12

Reply:
I am a bit confused in cidr block. /12 or /16 should be the default one?

Reply:
This should be a variable, and the default value can be 172.16.0.0/12 which is the full CIDR for the reserved IP space that docker is default configured to use.  The smaller CIDR you list above `172.18.0.0/16` will only work for 172.18.0.0 - 172.18.0.256, but some docker setups use 172.16.x.x or 172.17.x.x by default, and many people override their docker to use other reserved IP spaces such as 192.168.x.x or 10.x.x.x, so it's important to have a sensible default but allow for overrides as needed.